PokemonGo.RocketAPI/Client.cs

```diff
@@ -109,29 +109,52 @@ public Client(ISettings settings)
         public double CurrentAltitude { get; private set; }
 
         public async Task<CatchPokemonResponse> CatchPokemon(ulong encounterId, string spawnPointGuid, double pokemonLat,
-            double pokemonLng, MiscEnums.Item pokeball)
+            double pokemonLng, ItemId pokeball)
         {
-            var customRequest = new Request.Types.CatchPokemonRequest
+            CatchPokemonMessage catchPokemon = new CatchPokemonMessage()
```

Comment:
Those guidelines are violated throughout the codebase.

Plus, it doesn't explictly say to use var in those cases. Just that they only use var in those situations. Seems more of a standard of when not to use var than when to use var.